tests/load_variables_from_profiles.py

```diff
@@ -66,4 +67,9 @@ def get_variables(arg):
     variables['HOST_PROTOCOL'] = 'https' if variables['USE_HTTPS_FOR_TESTS'] else 'http'
     variables['MODEL_TEST_ENCLAVE'] = variables['ENCLAVES'][0] if len(variables['ENCLAVES']) > 0 else 'UNKNOWN_ENCLAVE'
 
+    if 'dex' in data and data['dex']['enabled'] and 'staticPasswords' in data['dex']['config'] and \
```

Comment:
We should move auth credentials to secrets

Reply:
Creds are moved to secrets